PowerShellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingServiceUtilities.cs

```diff
@@ -345,5 +346,18 @@ private void AddEventHandlers()
             _runspace.StateChanged += _runspace_StateChanged;
             _runspace.AvailabilityChanged += _runspace_AvailabilityChanged;
         }
+
+        /// <summary>
+        /// Invokes given script on the provided PowerShell object after setting its runspace object

```

Comment:
Comment to remind folks that the PowerShell object should be within a using type of scope, since it isn't included in here.  

Alternatively:  Extension method upon the PowerShell object?